backend/development.ini.sample

```diff
@@ -2,8 +2,18 @@
 # Tracim Config File #
 ######################
 
+## How work this file
+#
+# Tracim use all parameters not commented here (environnement variable is used first if define).
+# Lines who start by `;` is used to comment parameters and show default value available in Tracim code.
+# Lines who start by `#` is used for comment and to show parameters examples.
```

Comment:
Petite retouche des explications :

## Configuration
# Parameters can be defined via environment variables or in the config file.
# Environment parameters are used in priority if a variable is defined in both.

## File Syntax
# Comments start by `;` or `#`.
# In the sample config file, `;` commented line are used to indicate the default values used by Tracim;
# `#` commented lines are used for comments/examples.
# Boolean values can be `True` or `False`
# All parameters starting with `basic_setup` can be used as variables in other parameters. They are all mandatory.

Reply:
@grignards I do agree for almost all but basic_setup parameter are not really mandatory, it's just shortcuts to allow setup basic setup tracim parameters without needing to read the all configuration file. So we should probably remove the "mandatory" part.

Reply:
OK, I rewrote this part: `For simple needs they are all you need to customize`.

Reply:
+1:

Reply:
Is that ok with you if I push a commit with those changes ?

Reply:
ok, thanks.